woocommerce/woocommerce
# LYS - Delete WooPayments test orders on launch

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #47769 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new JN site with this branch.
2. Install and activate `WooPayments` plugin.
3. Install and activate `SQL Executioner` plugin.
4. Add a new product.
5. Add a new order with the product.
6. Copy the order I.D.
7. Go to `Tools -> SQL Executioner` and run the following SQL.

```
INSERT INTO `wp_wc_orders_meta` (`order_id`, `meta_key`, `meta_value`)
VALUES
	(YOUR-ORDER-ID-HERE, '_wcpay_mode', 'test');

```
8. Go to `WooCommerce -> Home` and click on `Launch your store`
9. Confirm 1 test order is found.
10. Toggle on remove test order and click on `Launch your store`
11. Go back to `Launch your store` task and confirm no test order is found.






### Changelog entry




<details>

#### Significance




#### Type




#### Message 

LYS - Add 'Remove test orders' for WooPayments

#### Comment 

</details>
